manage.py

```diff
@@ -312,3 +316,16 @@ def config(config_key):
 @app.cli.command()
 def set_celery_beat_healthy():
     return celery_beat_health_check()
+
+
+@app.cli.command()
+def healthcheck():
+    """Calls the healthcheck API"""
+    result = requests.get(
+        url_for('check')
+    )
+    print(json.dumps(result.json(), indent=4))
+
+    # Return success (0) if passing status code
+    # otherwise return fail (1)
+    return sys.exit(int(not result.ok))
```

Comment:
I don't think the exit code is working correctly:
```
$ flask healthcheck
{
    "status": "failure",
    "timestamp": 1541090957.455822,
    "hostname": "zinc",
    "results": [
        {
            "output": "Celery is not available.",
            "checker": "celery_available",
            "expires": 1541090966.440825,
            "passed": false,
            "timestamp": 1541090957.440825
        },
        {
            "output": "Celery beat is not available.",
            "checker": "celery_beat_available",
            "expires": 1541090966.442809,
            "passed": false,
            "timestamp": 1541090957.442809
        },
        {
            "output": "PostgreSQL is available.",
            "checker": "postgresql_available",
            "expires": 1541090984.454893,
            "passed": true,
            "timestamp": 1541090957.454893
        },
        {
            "output": "Redis is available.",
            "checker": "redis_available",
            "expires": 1541090984.4558,
            "passed": true,
            "timestamp": 1541090957.4558
        }
    ]
}
$ echo $?
0
```

Reply:
@CodeRhymesLife Sorry, on second thought this seems like the same `500`/`200` debate we had. I think it should only return non-`0` if there was an issue performing the check, but still `0` if unhealthy